pimd/pim_zlookup.c

```diff
@@ -298,23 +282,17 @@ static int zclient_read_nexthop(struct pim_instance *pim,
 				nbr = pim_neighbor_find_if(ifp);
 
 			if (nbr) {
-				nexthop_tab[num_ifindex].nexthop_addr.family =
-					AF_INET;
-				pim_addr_to_prefix(
-					&nexthop_tab[num_ifindex].nexthop_addr,
-					nbr->source_addr);
+				nexthop_tab[num_ifindex].nexthop_addr =
+					nbr->source_addr;
 			}
 			++num_ifindex;
 			break;
 		default:
 			/* do nothing */
 			{
```

Comment:
`{ }` now not needed anymore (was used before because `addr_str` variable declared)

(But I'm going to hit merge anyway in a second…)